apps/dashboard/app/views/batch_connect/sessions/card/_card_header.html.erb

```diff
@@ -0,0 +1,29 @@
+<%- 
+  session_class = status_context(session)
+  badge_class = "badge-#{session_class}"
+  alert_class = "alert-#{session_class}"
+  num_nodes = session.info.allocated_nodes.size
+  num_cores = session.info.procs.to_i
+-%>
+
+<div class="card-heading">
+  <div class="h5 card-header overflow-auto <%= alert_class %>">
```

Comment:
This changed from an actual `h5` to just the class h5 for accessibility reasons. The reason being that an h5 doesn't really make sense without h1-h4 headings also.